omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java

```diff
@@ -382,7 +382,7 @@ private boolean testOperationImplemented(OperationEnum operation, DelegatingReso
 						return false;
 					} else {
 						method.invoke(resourceHandler, RestConstants.SWAGGER_IMPOSSIBLE_UNIQUE_ID,
-						    RestConstants.SWAGGER_IMPOSSIBLE_UNIQUE_ID, new String(), new RequestContext());
+						    RestConstants.SWAGGER_IMPOSSIBLE_UNIQUE_ID,"", new RequestContext());
```

Comment:
```suggestion
						    RestConstants.SWAGGER_IMPOSSIBLE_UNIQUE_ID, "", new RequestContext());
```
Yes, I know, I'm picking on details. ;P